packages/edit-post/src/components/block-manager/index.js

```diff
@@ -63,6 +64,13 @@ function BlockManager( {
 						),
 						numberOfHiddenBlocks
 					) }
+					<Button
+						variant="secondary"
+						onClick={ () => enableAllBlockTypes( blockTypes ) }
+						label={ __( 'Enable all' ) }
```

Comment:
Is this `label` attribute necessary if the `<Button>` is wrapping text?

Reply:
I've seen this in other places in the code. However, searching more, most of the time this is not the case.
I removed the label.